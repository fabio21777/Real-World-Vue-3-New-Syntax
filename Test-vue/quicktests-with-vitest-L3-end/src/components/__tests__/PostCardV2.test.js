import { mount } from '@vue/test-utils'
import { describe, expect, test} from 'vitest'
import PostCardV2 from '../PostCardV2.vue'



describe('componente de card versÃ£o 2', () => {
    test('criar primeiro post', () => {
      const title = 'primeiro post'
      const body = 'conteudo do primeiro post'
      const wrapper = mount(PostCardV2, {
        props: { title, body }
      })
      expect(wrapper.html()).toMatchSnapshot()
    })
  })