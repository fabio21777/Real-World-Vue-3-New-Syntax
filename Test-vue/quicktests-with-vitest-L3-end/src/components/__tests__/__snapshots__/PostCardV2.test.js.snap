// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`componente de card versão 2 > created posts renders correctly 1`] = `
"<div>
  <h1>First Post</h1>
  <p>This is my first simple post</p>
</div>"
`;

exports[`componente de card versão 2 > criar primeiro post 1`] = `
"<div>
  <h1>primeiro post</h1>
  <p>conteudo do primeiro post</p>
</div>"
`;
